kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/MarkerRange.java

```diff
@@ -201,9 +201,9 @@ private Boundary() {
     }
 
     public Boundary(MarkerComparator comparator, Marker bound, boolean isInclusive) {
-      parquet.Preconditions.checkArgument(comparator != null,
+      Preconditions.checkArgument(comparator != null,
```

Comment:
Parquet doesn't want to depend on guava, so it defined its own precondition helpers. It was an accident to use the Parquet one here (or anywhere outside of Parquet).